lib/cookie.c

```diff
@@ -469,6 +469,8 @@ static int invalid_octets(const char *p)
 #define CERR_PSL           14 /* a public suffix */
 #define CERR_LIVE_WINS     15
 
+#define MAX_DATE_LENGTH 80
```

Comment:
Is 80 chars defined in any spec or is it a should-be-enough-for-everyone value? Regardless, it warrants a comment explaining the value.

Reply:
Thanks, good point. Will fix. It is "long enough for all standard formats and then some more just to play safe".